sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/StandardMetricsExtractionProcessor.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable // TODO: remove and fix errors
```

Comment:
Since this is a new class, please remove this line.

Reply:
will do a follow up on this, Still getting some errors.